sdk/src/wallet/error.rs

```diff
@@ -135,6 +136,13 @@ impl Serialize for Error {
     }
 }
 
+// Implements conversion from Infallible. This never happens, but is required by the compiler.
```

Comment:
What needs this? Feels like a signal that some other design is wrong.

Reply:
https://github.com/qrayven/iota-sdk/blob/d59e291c78f6ec3b51240656dcd662f37ad79fe6/sdk/src/wallet/account/operations/transaction/high_level/send_nft.rs#L32 This part (`Error=Into<crate::wallet::error`> This is necessary when we want to parametrize the error. In the case when infallible `NftID` is provided, the compiler uses the 'Infailable' enum, because it cannot recognize the error type. I could remove the `TryInto`, but we could lose the ability to use `str` as nft id

Reply:
Why are you using `TryInto` there rather than `Into`?

Reply:
because conversion from `str`  to  `NftID` is failable.  We don't have implementation `From<str>` for `NftId`, only `TryFrom`.

Reply:
Hm. I guess it's fine.